src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6292,4 +6292,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ConstOutOfRangeChecked_Title" xml:space="preserve">
     <value>Constant value may overflow at runtime (use 'unchecked' syntax to override)</value>
   </data>
+  <data name="ERR_NotOverridableAPIInRecord" xml:space="preserve">
+    <value>'{0}' disallows overriding and containing 'record' is not sealed.</value>
```

Comment:
>'record' [](start = 53, length = 8)

nit: Feels like we don't need the quotes on record

Reply:
> nit: Feels like we don't need the quotes on record 

Will Adjust in the next PR

---
In reply to: [451900246](https://github.com/dotnet/roslyn/pull/45647#discussion_r451900246) [](ancestors = 451900246)